calcom/cal.com
# fix: incorrect booking seats full error on collective seated event

## What does this PR do?



This issue occurs because the backend compares the total number of attendees (team members + booked seats) to the total number of seats per slot, instead of comparing the actual booked seats to the total number of seats per slot.

- Fixes #15600 (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)



https://www.loom.com/share/ff752121048b4e18a1ad64026963347d?sid=6686b5a4-d39f-4019-8122-c27630ae66db

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



1. Create a collective team seated event with a total of 3 seats.
2. Assign 3 team members to attend the event.
3. Book 1 seat for the event.
4. Attempt to book an additional seat for the same slot.
5. Notice the second booking is successful.
